app/src/main/java/org/breezyweather/theme/weatherView/materialWeatherView/MaterialPainterView.kt

```diff
@@ -228,21 +235,6 @@ class MaterialPainterView(
         background = getWeatherBackgroundDrawable(weatherKind, daylight)
     }
 
-    override fun onSizeChanged(w: Int, h: Int, oldw: Int, oldh: Int) {
-        super.onSizeChanged(w, h, oldw, oldh)
-
-        if (measuredWidth != 0 && measuredHeight != 0) {
-            val width = context.getTabletListAdaptiveWidth(measuredWidth)
-            val height = measuredHeight
-
-            if (canvasSize[0] != width || canvasSize[1] != height) {
-                canvasSize[0] = width
-                canvasSize[1] = height
-                setWeatherImplementor()
-            }
-        }
-    }
-
```

Comment:
I don't understand the changes in this file.

From what I read, Android doesn't know the real size at start and it needs to calculate it.
Once it is done, it will call `onSizeChanged`.
So why the change?

Reply:
This was my mistake, removing the `getTabletListAdaptiveWidth` call in `onSizeChanged` causes some odd visual effects because the width isn't remaining fixed. This change seemed to fix the problem, I can see this isn't actually the behavior I intended. I'll revert this change.